src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/abstractItems/AContainer.java

```diff
@@ -273,6 +274,23 @@ protected void tick(Block b) {
         }
     }
 
+    /**
+     * This method will remove charge from a location if it is chargeable.
+     *
+     * @param l location to try to remove charge from
+     * @return Whether charge was taken if its chargeable
+     */
+    protected boolean takeCharge(@Nonnull Location l) {
```

Comment:
Since this is a "protected" method, the Non-nullability of this parameter should also be enforced via `Validate.notNull(l, message)`